application-services/custom/fledge-export/Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: build clean
+
+GO=CGO_ENABLED=1 go
+
+# VERSION file is not needed for local development, In the CI/CD pipeline, a temporary VERSION file is written
+# if you need a specific version, just override below
+APPVERSION=$(shell cat ./VERSION 2>/dev/null || echo 0.0.0)
+
+# This pulls the version of the SDK from the go.mod file. If the SDK is the only required module,
+# it must first remove the word 'required' so the offset of $2 is the same if there are multiple required modules
+SDKVERSION=$(shell cat ./go.mod | grep 'github.com/edgexfoundry/app-functions-sdk-go v' | sed 's/require//g' | awk '{print $$2}')
+
+MICROSERVICE=fledge-export
+GOFLAGS=-ldflags "-X github.com/edgexfoundry/app-functions-sdk-go/internal.SDKVersion=$(SDKVERSION) -X github.com/edgexfoundry/app-functions-sdk-go/internal.ApplicationVersion=$(APPVERSION)"
+
+GIT_SHA=$(shell git rev-parse HEAD)
+
+build:
+	$(GO) build $(GOFLAGS) -o $(MICROSERVICE)	.
```

Comment:
All the example build the executable as `app-service` which is in the upper .gitignore.
https://github.com/edgexfoundry/edgex-examples/blob/master/application-services/custom/simple-filter-xml/Makefile#L6

You will need to update the readme to reference this new name.